packages/components/organisms/f-footer/test/component/f-footer.component.desktop.spec.js

```diff
@@ -173,16 +183,7 @@ describe('Desktop - f-footer component tests - @browserstack', () => {
     });
 
     forEach([
-        ['lu', 'lu-en'],
-        ['nl', '.nl'],
-        ['nz', '.nz'],
-        ['pl', '.pl'],
-        ['pt', '/pt'],
-        ['ro', '/ro'],
-        ['es', '.es'],
-        ['ch_ch', '.ch'],
-        ['ch_en', '/en'],
-        ['ch_fr', '/fr']])
+    ])
     .it('should display link for country code "%s" and redirect to correct URL ("%s")', (country, expectedUrl) => {
```

Comment:
These two tests were identical so I merged them

Reply:
I think you forgot to remove lines 170-185 here

Reply:
You still have empty `forEach([])` left with the duplicated act and assert sections

Reply:
Ah whoops